lib/components/T3CeFeloginLogin/T3CeFeloginLogin.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="fe-login">
+    <T3Form ref="form" :elements="elements" @submit="onSubmit" />
+  </div>
+</template>
+<script>
+import T3Form from '../../components/T3Form/T3Form'
+import baseCe from '../../mixins/component/baseCe'
+
+export default {
+  name: 'T3CeFeloginLogin',
+  components: {
+    T3Form
+  },
+  extends: baseCe,
+  props: {
+    data: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    elements () {
+      return this.data.form.fields.pages
+    }
+  },
+  methods: {
+    onSubmit ({ validator, form, formRef }) {
+      const data = new FormData(formRef)
```

Comment:
now when we send a post, API return fullpage response, but we want to get only current content element to easily find success/error messages, so that's why we have to pass additional params:
https://github.com/TYPO3-Initiatives/nuxt-typo3/blob/master/lib/components/T3CeFormFormframework/T3CeFormFormframework.vue#L77 
```js
  formData.append('responseElementId', this.id)
      formData.append('responseElementRecursive', 1)
```

Reply:
however api doesn't deliver clean `status` yet., we have only `message` - we can't rely on these fields to confirm if user is logged or not, so we have to wait for changes in [@typo3-iniative/headless](https://github.com/TYPO3-Initiatives/headless)

Reply:
I've added those two field to the `FormData`.
About the status you mentioned. I would - at least until the API gives us a proper response - print the messages we get above the form fields. What do you think about that?
When we have a status indication of some sort we could refactor that in proper i18n messages as done in the `T3CeFormFormframework`.

Reply:
we can use these messages for now 👍

Reply:
I've added the messages above the form.
Currently in simple `<p>` tags. Do you have any recommendations about what tags / classes we should use / apply to those messages?

Reply:
currently, this is enough, I suppose people will override the template